pyiron_workflow/files.py

```diff
@@ -71,6 +72,10 @@ def create_subdirectory(self, path):
     def create_file(self, file_name):
         return FileObject(file_name, self)
 
+    @property
+    def is_empty(self) -> bool:
```

Comment:
I'm not really sure, but isn't usually `is_xyz` a function?

Reply:
Hmm, to my mind it was a property, while `empty` is a method since it's ambiguous since it might be a verb.

Ah, you're right though, [pathlib.Path](https://docs.python.org/3/library/pathlib.html) makes all it's `is_*` things methods. I'll change it.